manifests/update.pp

```diff
@@ -56,13 +56,20 @@
   } else {
     $_refresh = true
   }
+  $apt_update_successful = @("BASH")
+    for try in {${apt::_update['tries']}..1}; do
+      timeout ${apt::_update['timeout'] ${apt::provider} update && exit 0
```

Comment:
```suggestion
      timeout ${apt::_update['timeout']} ${apt::provider} update && exit 0
```

Reply:
Yeah, that's a bad typo.
Also,  I forgot that Debian doesn't typically install bash as the default shell.
Thirdly, I forgot to escape my dollar-signs.

Fixed all three problems with a re-push.

@kenyon -- Please have another look.